### PR TITLE
Hans browse4 & browseHome

### PR DESCRIPTION
**Pull request by hanssy130**

### CODE REVIEW

File: front_end/Static/main_style.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+var deviceHeight = Dimensions.get('window').height;
 
 export default StyleSheet.create({
   class__container: {
     backgroundColor: 'rgba(255,255,255,0.7)',
     borderRadius: 20,
     marginBottom: 16,
     padding: '5%',
-    width: '100%',
+    width: (deviceWidth * 80) / 100,
```
[the remark this answers is not in the document]
Comment (hanssy130) [reply front_end/Static/main_style.js:12]: With just '100%', it was only doing part of the screen. I didn't understand why. This worked, however.

[the remark this answers is not in the document]
Comment (shlee2017) [reply front_end/Static/main_style.js:12]: width 100% makes it the same width the next outer view width is.

[the remark this answers is not in the document]
Comment (hanssy130) [reply front_end/Static/main_style.js:12]: image

The top half shows with `width: '100%'`.
The bottom half shows with `width: deviceWidth * 0.8`

[the remark this answers is not in the document]
Comment (shlee2017) [reply front_end/Static/main_style.js:12]: but this also changes some of my code

[the remark this answers is not in the document]
Comment (hanssy130) [reply front_end/Static/main_style.js:12]: What does it affect? Either way, I don't think that the card sizes should be the size of how long our strings are, as shown when using `width: '100%'`.

Hmm, maybe we can fix the "Start typing..." input size? What do you think?

File: front_end/Inbox/Message.js [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
-    View
-} from 'react-native';
+import {Text, TouchableHighlight, View} from 'react-native';
 
 import Icon from '../Utility/Icon';
-import styles from '../Static/style';
+import styles from '../Static/main_style';
 import inbox_style from '../Static/inbox_style';
 
 export default class Message extends Component {
```
Comment (xumaple) [inline front_end/Inbox/Message.js:8]: Is this a class for a single message, or all messages between two people? If the latter, perhaps a more suggestive name would be helpful.

File: front_end/Matching/MatchingHome.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  render() {
+    return (
+      <SafeAreaView style={styles.match__background}>
+        <View style={styles.screen__header} />
+        <View style={[styles.content__container, styles.content__centering]}>
+          <TouchableOpacity style={[styles.button, {margin: 30}]}>
+            <Text style={styles.text__header}>Browse</Text>
+          </TouchableOpacity>
+
```
Comment (alohakatiex) [inline front_end/Matching/MatchingHome.js:19]: for code consistency can we delete line 19?

File: front_end/Matching/MatchingBrowseTwo.js [hunk 4]
```diff
@@ -1,50 +1,55 @@
 //Matching-Browse-2
 import React, {Component} from 'react';
 import {
-    SafeAreaView,
-    View,
-    Text,
-    TextInput,
-    TouchableOpacity,
+  SafeAreaView,
```
Comment (alohakatiex) [inline front_end/Matching/MatchingBrowseTwo.js]: I noticed for MatchingBrowseOne.js  and QuestionList.js you compressed these libraries into one line – can we keep this as one line as well then?

Comment (alohakatiex) [reply front_end/Matching/MatchingBrowseTwo.js]: update: I believe @shlee2017 actually mentioned it'd be more readable if each library had its own line – could you change the other 2 files instead?

File: front_end/Inbox/Message.js [hunk 5]
```diff
@@ -1,52 +1,54 @@
...
+        onPress={this.onPressMessage}
+        underlayColor="lightgray">
+        <View>
+          <View style={inbox_style.align__row}>
+            <View style={{width: '10%'}}>
+              <Icon size={40} name={'profile'} />
+            </View>
+            <View style={{marginLeft: 16, width: '85%'}}>
+              <View
+                style={{flex: 1, flexDirection: 'row', alignItems: 'center'}}>
```
Comment (alohakatiex) [inline front_end/Inbox/Message.js:37]: can we move this styling (line 37) up with line 36?

Comment (xumaple) [reply front_end/Inbox/Message.js:37]: I think this is the linter's fault - the full line is more than 80 characters long so it changed this line into two lines.

File: front_end/Inbox/Message.js (hunk 2 above)
Comment (hanssy130) [reply front_end/Inbox/Message.js:8]: May you please suggest a better name? I was asked to change it to main_style from Eddy. Perhaps matching_style?

Comment (xumaple) [reply front_end/Inbox/Message.js:8]: I'm talking about the class. 

export default class Message ... {

File: front_end/Static/main_style.js [hunk 6]
```diff
@@ -1,15 +1,12 @@
 import {StyleSheet, Dimensions} from 'react-native';
```
Comment (shlee2017) [inline front_end/Static/main_style.js:1]: get rid of dimension too

File: front_end/Inbox/Message.js (hunk 2 above)
Comment (hanssy130) [reply front_end/Inbox/Message.js:8]: Changed Message to SingleMEssage on InboxOne.js and Message.js

File: front_end/Matching/MatchingBrowseTwo.js (hunk 4 above)
Comment (hanssy130) [reply front_end/Matching/MatchingBrowseTwo.js]: Disabled Prettier - Code formatter to do this.

**Pull request merged**